### PR TITLE
Update activations.py

### PR DESCRIPTION
**Pull request by rohanreddych**
improve documentation of sigmoid function, softmax, softplus and swish activation functions

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -159,6 +159,14 @@ def selu(x):
 @keras_export('keras.activations.softplus')
 def softplus(x):
   """Softplus activation function.
+  
+  Example Usage:
+  
+  >>> a = tf.constant([-20, -1.0, 0.0, 1.0, 20], dtype = tf.float32)
+  >>> b = tf.keras.activations.softplus(a) 
+  >>> b.numpy()
+  [2.0611537e-09, 3.1326166e-01, 6.9314718e-01, 1.3132616e+00, 2.0000000e+01]
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: You should not need the `.numpy()` line.

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -172,6 +180,13 @@ def softplus(x):
 @keras_export('keras.activations.softsign')
 def softsign(x):
   """Softsign activation function.
+  
+  Example Usage:
+  
+  >>> a = tf.constant([-20, -1.0, 0.0, 1.0, 20], dtype = tf.float32)
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: The `-20` and `20` values don't bring anything new to this example.

File: tensorflow/python/keras/activations.py [hunk 3]
```diff
@@ -185,12 +200,21 @@ def softsign(x):
 @keras_export('keras.activations.swish')
 def swish(x):
   """Swish activation function.
-
+  Swish is an activation function which returns `x*sigmoid(x)`, swish tends to work
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: Empty line above this one

File: tensorflow/python/keras/activations.py [hunk 4]
```diff
@@ -172,6 +180,13 @@ def softplus(x):
 @keras_export('keras.activations.softsign')
 def softsign(x):
   """Softsign activation function.
+  
+  Example Usage:
+  
+  >>> a = tf.constant([-20, -1.0, 0.0, 1.0, 20], dtype = tf.float32)
+  >>> b = tf.keras.activations.softsign(a)
+  >>> b.numpy()
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: You should not need the `.numpy()` line

File: tensorflow/python/keras/activations.py (hunk 3 above)
Comment (mihaimaruseac) [reply tensorflow/python/keras/activations.py]: Also, let's rewrite this to look like two different statements.

File: tensorflow/python/keras/activations.py [hunk 5]
```diff
@@ -274,6 +298,8 @@ def sigmoid(x):
 
   >>> a = tf.constant([-20, -1.0, 0.0, 1.0, 20], dtype = tf.float32)
   >>> b = tf.keras.activations.sigmoid(a)
+  >>> b.numpy()
+  [2.0611537e-09, 2.6894143e-01, 5.0000000e-01, 7.3105860e-01, 1.0000000e+00]
   >>> b.numpy() >= 0.0
   array([ True,  True,  True,  True,  True])
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: No need for the `.numpy()` line, also, let's remove the `>= 0.0` part of the docstring while we're here

File: tensorflow/python/keras/activations.py [hunk 6]
```diff
@@ -185,12 +200,21 @@ def softsign(x):
...
+  >>> b = tf.keras.activations.swish(a)
+  >>> b.numpy()
+  [-4.1223075e-08, -2.6894143e-01,  0.0000000e+00,  7.3105860e-01, 2.0000000e+01]
+  
   Arguments:
       x: Input tensor.
 
   Returns:
-      The swish activation applied to `x`.
+      The swish activation applied to `x`: `x*sigmoid(x)`
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: Forgot to end with period

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/activations.py (hunk 5 above)
Comment (mihaimaruseac) [reply tensorflow/python/keras/activations.py]: Please ignore the comments about the `.numpy()` calls

File: tensorflow/python/keras/activations.py (hunk 2 above)
Comment (rohanreddych) [reply tensorflow/python/keras/activations.py]: Removed the 20's

File: tensorflow/python/keras/activations.py (hunk 5 above)
Comment (rohanreddych) [reply tensorflow/python/keras/activations.py]: Removed the >=0.0 line, but do we need to put a line saying it produces output in the range of 0 to 1

File: tensorflow/python/keras/activations.py (hunk 3 above)
Comment (rohanreddych) [reply tensorflow/python/keras/activations.py]: Added empty line

**Comment by rohanreddych** [comment]
@mihaimaruseac **Off topic doubt**  what does `keras_export()` do ? Once take a look at this https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/preprocessing/image.py file only some of the functions or classes have proper documentation can we add examples and description for the rest of the functions at the bottom of the page?

File: tensorflow/python/keras/activations.py (hunk 5 above)
Comment (mihaimaruseac) [reply tensorflow/python/keras/activations.py]: Yes, please. Let's add that

File: tensorflow/python/keras/activations.py (hunk 3 above)
Comment (mihaimaruseac) [reply tensorflow/python/keras/activations.py]: I liked the comparison between this and the other activation function that you used to have. Can we add it back?

**Comment by mihaimaruseac** [comment]
`keras_export` is similar to `tf_export`. Both are exporting a Python function to the public API. For example

```python
tf_export("foo")
def foo_impl(arg, *args, **kwargs):
  """Some function and documentation."""
  pass
```

This means that TF API has a symbol `tf.foo` which when called results in calling the `foo_impl` function. `keras_export` exports to `keras` instead of to `tensorflow` (which becomes `tf` due to `import tensorflow as tf`). See https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/python/util/tf_export.py;l=400;drc=1cea2490cb1fb1e930694caa04c36a3049491535

The lines you mention are just exports for functions brought in from different modules. If the functions are implemented in Python, then documentation should be added to that Python function. If not, then we should create a wrapping function to wrap these symbols and then attach documentation to the wrapping function.

**Comment by rohanreddych** [comment]
@mihaimaruseac once review the changes

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -201,7 +201,12 @@ def softsign(x):
 def swish(x):
   """Swish activation function.
   
-  Swish is an activation function which returns `x*sigmoid(x)`. 
+  Swish activation function which returns `x*sigmoid(x)`.
+  It is a smooth, non-monotonic function that consistently matches 
+  or outperforms ReLU on deep networks, it is unbounded above and 
+  bounded below & it is the non-monotonic attribute that actually
+  creates the difference.
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]:
```suggestion
  bounded below.
```

**Review by mihaimaruseac — DISMISSED** [review]
One minor change but it's great. Thank you

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by gbaned** [comment]
@rohanreddych Can you please address Ubuntu Sanity errors? Thanks!

**Comment by rohanreddych** [comment]
@mihaimaruseac changed the output for `.numpy()` as the test was failing.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by mihaimaruseac** [comment]
There are conflicts with master branch. Can you rebase please?

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by mihaimaruseac** [comment]
I fixed the conflict manually

**Comment by gbaned** [comment]
@rohanreddych Can you please address Ubuntu Sanity errors? Thanks!

**Comment by rohanreddych** [comment]
@gbaned @mihaimaruseac it says `Internal CI build failed` what does it mean?

**Comment by mihaimaruseac** [comment]
The build failed. The `Details` link on the right of the build shows you the log of the build. Scroll to the end,

```
==== Summary of sanity check results ====
1. do_configure_test: Run ./configure
  PASS
2. do_pylint: Python 3 pylint
  FAIL
3. do_check_futures_test: Check that python files have certain __future__ imports
  PASS
4. do_buildifier: buildifier check
  PASS
5. do_bazel_nobuild: bazel nobuild
  PASS
6. do_bazel_deps_query: bazel query
  PASS
...
```

which means pylint failed. Download the log, look for the pylint section and there is a list of pylint errors:

```
FAIL: Found 1 non-whitelisted pylint errors:
tensorflow/python/keras/activations.py:296: [C0301(line-too-long), ] Line too long (82/80)
```

**Review by mihaimaruseac — APPROVED** [review]

**Comment by gbaned** [comment]
@yifeif  Can you please take a look on cla/google test failure? Thanks!

**Comment by mihaimaruseac** [comment]
Fixed cla by playing with the CLA labels. Now this should be imported internally.
